User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+
+### Suggest duplicates
+
+You may not have been the first person to ask this question! Duplicates aren't necessarily a bad thing, but for people looking for information it's helpful to have links to any previous posts on the same topic. 
+
+### Downvotes
+
+Downvotes are the negative analogue to upvotes. They are used to reduce the visibility of low quality contributions and are used in the calculation of whether or not a particular contribution is well-received. 
+
+### Vote-to-lock
+
+Some contributions need serious help. They might be off-topic, or unclear without additional information from the original poster. In these cases the best thing to do is often to lock the post so it's no longer able to accept responses until the original poster updates his work. All lock votes are based on certain categories, each associated with a linked community Meta post. A certain threshold of lock votes vs. no-lock votes must be met before a post is locked, and editing the post to adhere to site guidelines will allow it to be unlocked. 
+
+### Direct edits
+
+Sometimes a user isn't responsive or their post needs some help or it will be closed. In this case it's possible for the higher trust level users to make changes directly instead of just suggesting. 
+ 
+### Review flags and suggested edits
+
+Flags and suggested edits go into a moderation queue for approval. Higher trust level users can review these and take action if necessary. 
+
+### Create tags
+
+Creating tags is a difficult task, as they need to be generic enough to be commonly chosen, while also not duplicating other existing tags. Currently this functionality is limited to users with enough experience to have a good idea of what to look for in a tag. 
+
+### Protect posts
+
+If a post is controversial or has attracted low quality answers, sometimes the moderation team will restrict answering to users with a higher trust level.
+
+### Temporarily lock posts
+
+Similar to post protection, sometimes the moderation team needs to lock a post to keep discussion from getting out of hand, or to give the original poster a chance to add more information. 
+
+### Soft-delete posts
+
+A soft deletion hides posts from general view. Used primarily for inappropriate topics and spam where locking is insufficient.
+ 
+### Soft-delete comments
+
+Same as soft-deletion for posts. 
+
+### Permanently lock posts
+
+Locking a post means no further edits or responses can be made. 
+
+### Suspend specific user permissions
+
+If any of the privileges on this list are being abused, moderators can remove any and all of them. Primarily used to combat spammers or individuals abusing the site. 
+
+### Sent private messages to users
+
+Sometimes moderators need to reach out to users directly if they seem to be having issues, are in a situation that requires personal information, or anything else best handled in confidence. 
```

Comment:
Let's steer away from the "requires personal information" part; we're trying to protect PII as much as possible.  I think it's sufficient to say "Sometimes moderators need to reach out to users confidentially", period.